app/uk/gov/hmrc/economiccrimelevyaccount/views/AccountView.scala.html

```diff
@@ -30,5 +34,101 @@
+                <span class="font-xsmall"></span>
+                <p class="card-sub-heading bold govuk-body-m">
+                    <span class="font-xsmall"></span>
+                </p>
+            </div>
+            <div class="card-action">
+                <ul class="govuk-list">
+                    <li>
+                        <a id="view-your-financial-overview" class="govuk-link" href="tbc">@messages("account.submitEcl")</a>
+                    </li>
+                    <li>
+                        <a id="past-payments-and-charges" class="govuk-link" href="tbc">@messages("account.howToComplete")</a>
+                    </li>
+                    <li>
+                        <a id="past-payments-and-charges" class="govuk-link" href="view-or-amend-returns">@messages("account.viewReturns")</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="card" id="aft-overview">
+            <!-- PAYMENTS CARD -->
+            <div class="card-body">
+                <h2 class="card-heading govuk-heading-m">
+                    @messages("account.payments.card.title")
+                </h2>
+                <p class="card-sub-heading bold govuk-body-m">
+                    <span class="font-xsmall">@messages("account.payments.subHeading")</span></p>
+                <p class="card-sub-heading bold govuk-body-m">
+                    <span class="font-xsmall"></span>
+                </p>
+            </div>
+            <div class="card-action">
+                <ul class="govuk-list">
+                    <li>
+                        <a id="aftContinueInProgressLink" class="govuk-link" href="tbc">@messages("account.payments.makePayment")</a>
+                    </li>
+                    <li>
+                        <a id="aftLoginLink" class="govuk-link" href="tbc">@messages("account.payments.setDirectDebit")</a>
+                    </li>
+                    <li>
+                        <a id="aftAmendLink" class="govuk-link" href="tbc">@messages("account.payments.viewHistory")</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+
 
+        <div class="card" id="event-reporting">
+            <!-- HISTORY CARD -->
+            <div class="card-body">
+                <h2 class="card-heading govuk-heading-m">
+                    @messages("account.history.card.title")
+                </h2>
+            </div>
+            <div class="card-action">
+                <ul class="govuk-list">
+                    <li>
+                        <a id="aftContinueInProgressLink" class="govuk-link" href="tbc">@messages("account.history.viewHistory")</a>
+                    </li>
+                    <li>
+                        <a id="aftLoginLink" class="govuk-link" href="tbc">@messages("account.history.viewReturns")</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
```

Comment:
I think the plan was to add these cards as part of the other tickets - however not an issue if you've added them already we may as well leave them.

As with my other comment though I would definitely define a `Card.scala.html` component that you can re-use

Reply:
Yeah, I would leave them in, they just fit in perfectly, It looks better, any logic around them can be added in those later tickets.